src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx

```diff
@@ -340,9 +340,7 @@ const OrderFulfillPage: React.FC<OrderFulfillPageProps> = props => {
                               );
                             }
 
-                            const availableQuantity =
-                              warehouseStock.quantity -
-                              warehouseStock.quantityAllocated;
+                            const availableQuantity = warehouseStock.quantity;
```

Comment:
Instead of ignoring `quantityAllocated` we should subtract from it the quantity allocated for this order